internal/commands/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,21 @@
+package kubernetes
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+)
+
+// BaseKubernetesCommand creates the base "kubernetes" command
+func BaseKubernetesCommand() commands.Command {
+	return &kubernetesCommand{
+		commands.New("kubernetes", "Manage Kubernetes clusters"),
```

Comment:
Should we add some sort of experimental note here? E.g., our Terraform provider docs have: `NOTE: this is an experimental feature in an alpha phase, the resource definition will change in the future.`.

Reply:
Go SDK seems to use `(EXPERIMENTAL)` at the end of the doc comment

Reply:
Fixed in https://github.com/UpCloudLtd/upcloud-cli/pull/190/commits/2be93a4a7d71bdec90e718c80c5c0842082ab6c3